config.yml

```diff
@@ -5,12 +5,12 @@
 # APPS_CHAINLINK_VERSION=1.0.0
 
 network: "ethereum_geth" # Selected network for test execution
-keep_environments: Never # Options: Always, OnFail, Never
+keep_environments: Always # Options: Always, OnFail, Never
```

Comment:
This should be left as Never by default so we don't accidentally leave a bunch of environments laying around.